### ISSUE TITLE
Minor revisions

### ISSUE THREAD

**Issue by asmeurer** [sympy/sympy-paper#215]
The reviews are back for minor revisions:

**Editor comments**
- [x] 1. In section 2.3, lines 128–129, it is written
  
  For instance, the identity √t2 = t holds if t is nonnegative (t ≥
  0). However, for general complex t, no such identity holds.
  
  The first sentence doesn’t make sense unless √· is defined; obviously it
  is intended to yield the nonnegative square root. The second sentence is
  incorrect, so needs rewording or deleting. For complex t it is true that t2 = t holds if t lies in the right half-plane, assuming √· is defined to be the square root lying in the right half-plane.
- [x] 2. Page 7, line 186: please state in the text whether this is typed as two lower case letter o’s, as opposed to being some special symbol.
- [x] 3. Page 10, line 315: replace “solving” by “finding its zeros”.
- [x] 4. Page 18, line 684: replace “is as performant as” by “performs as well as”. The former is not correct English.
- [x] 5. Reference [42]: the title and publisher appear to have run together.
- [x] 6. Supplement, line 1: delete “for”.
- [x] 7. Supplement, line 18: “dependends” → “depends”.

**Reviewer 1**

(there do not appear to be any comments from reviewer 1)

**Reviewer 2**

Basic reporting
No comments.

Experimental design
No comments.

Validity of the findings
No comments.

Comments for the author
This is a great paper. Thank you for addressing my concerns.

**Reviewer 3**

Basic reporting
The structure of the paper and the supplement is now improved, compared to the original submission, and the materials are now much easier to read and comprehend.

Experimental design
No Comments

Validity of the findings
No Comments

Comments for the author
The paper
- [x] 1. The abstract and the paper itself mention the paper's supplement in several places. I believe it would be beneficial to add a short paragraph, possibly after line 90, that would list the supplement's contents in a manner similar to that of the paper's contents (lines 83-90), with the list submodules, to make it easier for an interested reader to find their way around it.
- [x] 2. In line 323, it should say "...meaning that the entries...".
- [x] 3. In line 673, it should say either "poorer than that of its commercial competitors" or "poorer than its commercial competitors'" (ending in apostrophe).
- [x] 4. In line 813, one "PhD thesis" seems to be superficial.

The supplement
- [x] 5. Figure 1 is hard to read. I suggest putting the labels on the outside of the loops.
- [x] 6. Figure 2 has the wrong link towards SymPy Gamma's computation of tan(x) instead of ∫egrate(tan(x)).

**Comment by asmeurer** [comment]
Some thoughts:

Editor comment 1: I guess we need to reword this somehow. I think the editor is misunderstanding our use of the term "identity". An equation A = B is an identity on some domain (say, for complex valued variables) if it holds for _all_ values in that domain. Saying `sqrt(t**2) = t` is an identity for complex `t` because it holds for some complex `t` doesn't make sense. Any suggestions on how to reword this? I do want to leave it in, as it's the #1 (by far) real world example of the assumptions being needed to simplify an expression.

Review comment on the supplement figure 1: The figure code is generated automatically from the category theory module. I guess this should be made clearer in the caption (like "A diagram typeset in Xy-pic automatically by `XypicDiagramDrawer`").

Reviewer comment for supplement figure 2: Both links will show the integral, but it's probably better to actually use the integral link, as that's less confusing. 

The rest of the changes are straightforward.

**Comment by certik** [comment]
Regarding Editor comment 1, I propose a fix that I pushed into the `peer-reviews` branch in ceb444000109e4bda80c846ab9e52cadcb24b1de. The way it was written was indeed confusing, as the word identity was used both for an equation that only holds for some `t` as well as for an equation that holds for all complex `t`.

**Comment by asmeurer** [comment]
I see. I didn't notice that we used the word "identity" the first time. That is indeed wrong. I like your change (although see my comment), but it doesn't address the "how do you define square root" issue. Should we add a footnote indicating that "\sqrt{t} is the principle square root as usually defined, with a branch cut on the negative real axis" or something similar?

**Comment by certik** [comment]
@asmeurer I've indicated that a bit in the footnote, but you are right that I should also indicate that that identity assumes the sqrt is itself defined on the usual principal branch. I clarified it in ece7a3fa4cec511a8e66e235c43a97d29674aa06.

Regarding how sympy does it ---- does sympy assume the usual principal branch? I would think so, since that's what's used when you evaluate numerically. As well as in Python and other languages.
